php/src/ValueObjects/Security/SharableItem.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Kiniauth\ValueObjects\Security;
+
+use Kiniauth\Traits\Account\AccountProject;
+use Kiniauth\Traits\Security\Sharable;
+use Kinikit\Core\Exception\WrongParameterTypeException;
+
+/**
+ * Used for returning sharable items for display purposes etc.
+ */
+class SharableItem {
+
+    /**
+     * @var string
+     */
+    private $itemTypeLabel;
+
+    /**
+     * @var string
+     */
+    private $itemLabel;
+
+    /**
+     * @var string
+     */
+    private $sharerName;
+
+    /**
+     * @var string
+     */
+    private $sharerLogo;
+
+
+    public function __construct(mixed $sharable) {
+        if (!in_array(Sharable::class, class_uses(get_class($sharable)))) {
+            throw new WrongParameterTypeException("Sharable Items must be constructed with sharable objects");
+        }
```

Comment:
`class_uses` doesn't return traits of superclasses. We may need recursive function here.

Reply:
That's a good point and probably bears sorting - I think I wrote a recursive function for this